src/Routes/RosPage/ros-page.scss

```diff
@@ -85,3 +85,9 @@
     background: var(--pf-global--palette--red-200);
   }
 }
+
+.blue-300{
```

Comment:
```suggestion
.chart-blue-200 {
```

Reply:
Hello @karelhala - any suggestions on this topic?
I would like to know any best practice that other services follow when a variable except global color var is not accessible.

Code reference - https://github.com/RedHatInsights/ros-frontend/pull/60/files#diff-8f1db47e612613230614c86f8f671e0bd927e0fc9c01109bc4dbc5b7e6b634c3R70
Variable used in mockup page `--pf-chart-color-blue-200` i.e  #519DE9

I guess, I am over thinking but can existing classes be overridden with this import?

Reply:
That's a good question, the problem here is that PF has a color pallete defined by all of those CSS variables, but in the mocks you have the designer used color which is used in charts. These colors are not exported to global pallete unless you import anything from charts. You have a few options here
* contact the designer and discuss if the `var(--pf-global--palette--blue-300);` can be used instead of chart color (I'd prefer this solution)
* use the sass variable (clean CSS solution - prefered as a fallback if colors from default palette can't be used)
* use token value (clean JS solution - be carefull when importing from tokens, you have to use direct imports)
* use hex value provided in mocks (fallback if nothing from above works)

Reply:
Good discussion here. Using var(--pf-global--palette--blue-300)  sounds good to me!